src/views/ModelPlan/TaskList/ITSolutions/index.tsx

```diff
@@ -58,10 +58,10 @@ const ITSolutions = () => {
             </Route>
 
             <Route
-              path="/models/:modelID/task-list/it-solutions/:operationalNeedID/update-status"
+              path="/models/:modelID/task-list/it-solutions/:operationalNeedID/update-status/:solutionId"
```

Comment:
Within the tracker it's possible to update all solutions at once.  We need the ability to have the `solutionId` optional in this case.  If there is no solution id, and we are updating, we need to render all the solutions and have the text reflective of the update.

The flow to update all the solutions is as follows: 
From the tracker, click 'View Details', then in the solution details, click 'Update status and timing'.  Then you will comfirm the solutions on the next page and click 'Continue'.  Then you are taken to the update-status view where you should be able to update all the solutions